src/client/components/serializer/MarkdownRenderer.js

```diff
@@ -138,10 +150,14 @@ const NodeSerialize = {
     const columnsCount = obj.nodes.get(0).nodes.size;
     return `${children}|${new Array(columnsCount).fill(' --------- |').join('')}`;
```

Comment:
Should we use `Array.prototype.fill`? It not supported in IE according to [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/fill)
In any case we already need some polyfill for `Symbol` data type. If it a babel-polyfill, it implements `Array.prototype.fill` method.
Need @asergeev-sc approval.